rules/gdi-structures.yaml

```diff
@@ -405,8 +405,8 @@ GAARMORY:
 		OrderName: ArtilleryStrikePowerInfoOrder
 		Prerequisites: garadr, upgrade.gdf
 		Icon: artystrike
-		ChargeTime: 270
-		QuantizedFacings: 8
+		ChargeTime: 330
+		QuantizedFacings: 2
```

Comment:
Predicable abilities have a small advantage for the attacker and a big advantage for the defender. I don't really see the point of doing this change but let's try it.